pkg/processor/build/runtime/nodejs/test/nodejs_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"testing"
+
+	"github.com/nuclio/nuclio/pkg/processor/build/runtime/test/suite"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSuite struct {
+	buildsuite.TestSuite
+}
+
+func (suite *TestSuite) SetupSuite() {
+	suite.TestSuite.SetupSuite()
+
+	suite.TestSuite.RuntimeSuite = suite
+}
+
+func (suite *TestSuite) GetFunctionInfo(functionName string) buildsuite.FunctionInfo {
+	functionInfo := buildsuite.FunctionInfo{
+		Runtime: "nodejs",
+	}
+
+	switch functionName {
+
+	case "reverser":
+		functionInfo.Path = []string{suite.GetTestFunctionsDir(), "common", "reverser", "nodejs", "handler.js"}
+		functionInfo.Handler = "handler"
+
+	default:
```

Comment:
Please implement `json-parser-with-function-config` and `json-parser-with-inline-function-config` as well - all runtimes should implement this to make sure nothing runtime-specific is broken in the build.

Reply:
Will do.

Reply:
Done